queries/inko/highlights.scm

```diff
@@ -190,6 +190,9 @@
   name: _ @variable.parameter)
 
 (call
-  name: _ @function)
+  name: [
+    (name)
+    (constant)
+  ] @function)
```

Comment:
The only possible nodes here are `name`, `constant` and `integer`, and that is likely to remain the case for a long time.